utils/psh-applets/wget.md

```diff
@@ -0,0 +1,29 @@
+# wget
```

Comment:
Please also change the headers of the existing descriptions to lowercase for consistency. Otherwise LGTM